modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -152,23 +289,27 @@ const CustomMenu = () => {
 };
 ```
 
-#### Layout and Column
+🤖 The codemod will rename `Menu`, `MenuProps`, `MenuState`, `MenuItem` and `MenuItemProps` to their
+deprecated names.
 
-We are [soft deprecating](#soft-deprecation) `Layout` and `Column`. It has been renamed to
-`DeprecatedLayout` and `DeprecatedColumn`. Similarly, `LayoutProps` has been renamed to
-`DeprecatedLayoutProps`, `Column` to `DeprecatedColumn` and `ColumnProps` to
-`DeprecatedColumnProps`. You may continue to use this component, but note that we plan to
-[hard-deprecate](#hard-deprecation) it in Canvas Kit v9.
+You may continue to use this component, but please note that we plan to
+[hard-deprecate](#hard-deprecation) it in v9. Consider using the [Menu](/components/popups/menu/) in
+the Main package instead.
 
-This components can be replaced by the new `Grid` component existing in
-`@workday/canvas-kit-react/layout`.
+---
 
-🤖 The codemod will rename `Layout`, `LayoutProps`, `Column` and `ColumnProps` to their deprecated
-names:
+### Layout and Column
+
+We've [soft-deprecated](#soft-deprecation) `Layout` and `Column` and renamed the following items:
+
+- `Layout` to `DeprecatedLayout`
+- `LayoutProps` to `DeprecatedLayoutProps`
+- `Column` to `DeprecatedColumn`
+- `ColumnProps` to `DeprecatedColumnProps`
 
 ```tsx
 // v7
-import {Layout, Column} from '@workday/canvas-kit-react/drawer';
+import {Layout, Column} from '@workday/canvas-kit-react/layout';
```

Comment:
Should we use `Column` instead of `Layout.Column` in the code snippets here? I understand both work, but it may make more sense to use `Column` within the context of the deprecations we're showing here (i.e., `Column` to `DeprecatedColumn`).

Reply:
Handle this in a follow-up PR.